.github/workflows/clean_dedicated_host.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: Clean Dedicated Host Daily Resources Cleaner
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Run Every Day At Midnight
+  workflow_dispatch:
+
+jobs:
+  clean-dedicated-hosts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.TERRAFORM_AWS_ACCESS_KEY_ID }}
```

Comment:
I didn't realize/notice that GH actions outside of the integration tests use these secrets. I think it warrants consolidating a README that just outlines the different secrets needed to run things on your fork. Not a blocker for this PR, and I'm more than willing to follow up with a PR for that after this gets merged.